core/src/components/textarea/test/autogrow/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
```

Comment:
A lot of the other tests seem to have visual diffs too. Is this caused the `--padding-bottom: 8px` addition?